### DISCUSSION TITLE
How can I  add more layers in the current version :((

### DISCUSSION THREAD

**Discussion by xntzmk** [vial-kb/vial-gui#194] [Q&A]
I've checked the issue for similar questions but found the pyconfig.h file instead of config.h file. But there is no #define DYNAMIC_KEYMAP_LAYER_COUNT

**Comment by xyzz** [comment]
The config.h file is a part of the vial-qmk repository, located under the vial keymap for your keyboard. To increase the number of layers, add this line `#define DYNAMIC_KEYMAP_LAYER_COUNT XX` where XX is the desired number. You'll need to recompile the firmware for your keyboard to increase the number of layers. See here for an example https://github.com/vial-kb/vial-qmk/blob/a0df3e2488a9ad2bc93a37e482e6b3fedf6e3f59/keyboards/mini49/keymaps/vial/config.h#L11
